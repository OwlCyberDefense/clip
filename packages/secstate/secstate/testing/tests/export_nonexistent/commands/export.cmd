/usr/sbin/secstate export fjdskafdjskl zcxjzcnjkl
