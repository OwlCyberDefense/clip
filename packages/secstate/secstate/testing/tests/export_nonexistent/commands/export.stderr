No benchmark 'fjdskafdjskl' has been imported
